solutions/control-flow/predict-the-seat.js

```diff
@@ -0,0 +1,24 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, "utf-8");
+let idx = 0;
+data = data.split("\n");
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+let t = parseInt(readLine());
+let seat = ["L", "M", "U", "L", "M", "U", "SL", "SU"];
```

Comment:
And instead of using array, and fetching the berth using index, can we use if & else if conditions, like below
if (seatNumber > 0 && seatNumber <= numOfBerths) {
   const berth = seatNumber % 8
   if (berth == 0){
	   console.log("SU")
   } else if (berth == 1 || berth == 4){
	   console.log("L")
   }else if (berth == 2 || berth == 5){
	   console.log("M")
   }else if (berth == 3 || berth == 6){
	   console.log("U")
   }else if (berth == 7){
	   console.log("SL")
   }
  } else {
    console.log("Invalid");
  }